src/assets/toolkit/styles/utils/text.css

```diff
@@ -16,6 +16,16 @@
   font-weight: var(--font-weight-semibold) !important;
 }
 
+
+/**
+ * Text color
+ */
+
+.u-textWhite {
+  color: var(--color-white);
+}
+
```

Comment:
Doesn't relate to photo change, but I had removed it from my WIP stylesheet in the last PR, and it was being used by the header text, since the light purple doesn't have enough contrast.